gateway/src/task.rs

```diff
@@ -161,7 +161,8 @@ pub fn restart(project_id: Ulid) -> impl Task<ProjectContext, Output = Project>
             .container()
             .and_then(|container| ProjectCreating::from_container(container, 0).ok())
             .unwrap_or_else(|| {
-                ProjectCreating::new_with_random_initial_key(ctx.project_name, project_id, 1)
+                // idle minutes 1 is generally buggy, therefore use 3.
+                ProjectCreating::new_with_random_initial_key(ctx.project_name, project_id, 3)
```

Comment:
Hmm, can you remind me why we set idle minutes like this here?

Reply:
I think P set it like that since we know the project is about to get deleted. I'm thinking there's a small chance the 1 minute setting (which acts like 0 minutes) could start conflicting when state machine is slow (delete command trying to start/stop + ambulance trying to idle).

Reply:
Ah, I see, thanks!